src/BDCS/API/V0.hs

```diff
@@ -1271,14 +1271,17 @@ recipesFreeze ServerConfig{..} mbranch recipe_names = liftIO $ RWL.withRead (git
     getVersionFromNEVRA :: PackageNEVRA -> String
     getVersionFromNEVRA nevra = T.unpack $ T.concat [epoch $ pnEpoch nevra, pnVersion nevra, "-", pnRelease nevra]
       where
-        epoch (Just e) = e `T.append` ":"
         epoch Nothing  = ""
+        epoch e = do
```

Comment:
This can be simplified to:

    epoch (Just e) = (T.pack $ show e) T.append` ":"